templates/conf-mesos.j2

```diff
@@ -8,6 +8,15 @@ LOGS={{mesos_log_location}}
 ULIMIT="{{mesos_ulimit}}"
 ZK="{{ mesos_zookeeper_masters }}"
 
+{% if mesos_ssl_enabled %}
+export LIBPROCESS_SSL_ENABLED=true
```

Comment:
I have it currently working using exactly this.
If you like I can check tomorrow if the MESOS_ prefix also works?